auslib/web/public/api.yml

```diff
@@ -587,3 +587,210 @@ paths:
       responses:
         200:
           description: version.json content.
+
+# Public Rules endpoints
+  /rules:
+    get:
+      operationId: "auslib.web.public.rules.get"
+      description: ""
+      produces: ["application/json"]
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/RulesWrapper"
+
+  /rules/{id_or_alias}:
+    get:
+      operationId: "auslib.web.public.rules.get_by_id_or_alias"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: id_or_alias
+          in: path
+          required: true
+          type: string
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/Rule"
+
+  /rules/{rule_id}/revisions:
+    get:
+      operationId: "auslib.web.public.rules.get_revisions"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: rule_id
+          in: path
+          required: true
+          type: integer
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/RuleHistoryWrapper"
+
+  /releases:
+    get:
+      operationId: "auslib.web.public.releases.get"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: product
+          in: query
+          required: false
+          type: string
+        - name: name_prefix
+          in: query
+          required: false
+          type: string
+        - name: names_only
+          in: query
+          required: false
+          type: string
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/RulesWrapper"
```

Comment:
You right, I opened this PR mainly for validade the access to admin api, from public api. There are a lot of details to be implemented yet.